node/resolvers/checkout/index.ts

```diff
@@ -4,9 +4,30 @@ import httpResolver from '../httpResolver'
 import paths from '../paths'
 import paymentTokenResolver from './paymentTokenResolver'
 
+/**
+ * It will convert an integer to float moving the
+ * float point two positions left. That is needed
+ * once the OrderForm REST API return an integer
+ * colapsing the floating point into the integer
+ * part.
+ * 
+ * @param int An integer number
+ */
```

Comment:
I think that a good doc will be something like: 
 
```
/**
 * Convert an integer to float moving the
 * float point two positions left. 

 * We needed to make a convention of the product 
 * price on different API's. Once the Checkout API 
 * returns an integer instead of a float, and the 
 * Catalog API returns a float. 
**/
```